src/test/java/org/broadinstitute/consent/http/resources/ConsentResourceTest.java

```diff
@@ -131,7 +130,7 @@ public void testMissingDataUseUpdate() throws Exception {
         initResource();
 
         Response response = resource.update(consent.getConsentId(), consent, user);
-        assertEquals(400, response.getStatus());
+        Assertions.assertEquals(400, response.getStatus());
```

Comment:
can we keep it like before, where it's like `assertEquals` instead of `Assertions.assertEquals`? I totally understand if it's a big lift cause there are so many changes, but it seems unnecessarily verbose.

Reply:
I'll look at it ... this was the default Intelli-j conversion.